serde_v8/error.rs

```diff
@@ -57,3 +68,90 @@ impl serde::de::Error for Error {
     Error::Message(msg.to_string())
   }
 }
+
+pub(crate) fn value_to_type_str(value: v8::Local<v8::Value>) -> String {
```

Comment:
```suggestion
pub(crate) fn value_to_type_str(value: v8::Local<v8::Value>) -> &'static str {
```

remove `to_string` in the end.